### PR TITLE
doc: formatting, grammar, and some minor changes

### PR DESCRIPTION
**Pull request by CodeDotJS**
## Description

Was going through the documentation and found some minor fixes related to grammar, formatting etc on different pages.

## Type of change

- [x] This change requires a documentation update

### CODE REVIEW

File: docs/tutorials/drm-config.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ playback.  Passing in a higher security level than can be supported will cause
 default is the empty string, which is the lowest security level supported by the
 key system.
 
-Each key system has their own values for robustness.  The values for Widevine
+Each key system has its values for robustness.  The values for Widevine
```
Comment (theodab) [inline docs/tutorials/drm-config.md:148]: I don't quite agree with this particular change. "their own" better conveyed the idea the key systems have DIFFERENT sets of robustness values. Perhaps "its own"?

File: docs/tutorials/faq.md [hunk 2]
```diff
@@ -123,9 +123,9 @@ configuration][buffering] and the docs for the
 [`.streaming.bufferingGoal`][StreamingConfiguration] configuration.
 
 Another factor is the segment size.  It may take up to 2 segments before Shaka
-Player has enough information to form a bandwidth estimate and make a decision.
+The Player has enough information to form a bandwidth estimate and make a decision.
```
Comment (theodab) [inline docs/tutorials/faq.md:126]: This isn't the beginning of a sentence. This would result in "Shaka The Player".

File: docs/tutorials/faq.md [hunk 3]
```diff
@@ -202,12 +202,12 @@ supported.  This supports both DASH and HLS manifests.
 **Q:** The Nightly Demo isn't loading for me!
 
 **A:** Are you looking at the uncompiled build with an AdBlocker enabled?
-We're rolling out ad support which is triggering some ad blockers to block
+We're rolling out ad support which is triggering some adblockers to block
```
Comment (theodab) [inline docs/tutorials/faq.md:205]: Hm. Is "adblocker" considered a word now? Like, the [wikipedia article on the subject](https://en.wikipedia.org/wiki/Ad_blocking) mostly keeps it at two words.

File: docs/tutorials/license-server-auth.md [hunk 4]
```diff
@@ -190,7 +190,7 @@ Since v2.1.0, we support asynchronous filters.  This allows you to pause a
 license request, make an additional request for an authorization token, then use
 that token to complete the license request.
 
-Any filter that returns a Promise is an asynchronous filter.  NetworkingEngine
+Any filter that returns a Promise is asynchronous filter.  NetworkingEngine
```
Comment (theodab) [inline docs/tutorials/license-server-auth.md:193]: No, the "an" was grammatically correct there.

File: docs/tutorials/manifest-parser.md [hunk 5]
```diff
@@ -157,14 +157,14 @@ Media segment times are all in terms of the presentation timeline.  So if the
 content was originally multi-period, the period start time must be accounted for
 in the reference's `timestampOffset` field.
 
-First we ask for the index that corresponds with a start time.  Then on update,
+First, we ask for the index that corresponds with a start time.  Then on the update,
```
Comment (theodab) [inline docs/tutorials/manifest-parser.md:160]: "on the update" kind of suggests that the manifest will only ever update once. I'd leave that phrasing as-is.

File: docs/tutorials/manifest-parser.md [hunk 6]
```diff
@@ -346,8 +346,8 @@ MyManifestParser.prototype.loadReference_ =
 
 If your content is encrypted, there are a few changes to the manifest you need
 to do.  First, for each Stream that contains encrypted content, you need to set
-`stream.encrypted` to true and put the key IDs that the stream is encrypted with
-in the `stream.keyIds` set.  Filling out the `keyIds` is technically optional,
+`stream.encrypted` to true and put the key IDs that the stream is encrypted within
```
Comment (theodab) [inline docs/tutorials/manifest-parser.md:349]: No, "with in" was correct. You are taking the "key IDs that the stream is encrypted with" and putting them "in the `stream.keyIds` set".

File: docs/tutorials/network-and-buffering-config.md [hunk 7]
```diff
@@ -31,7 +31,7 @@ So, for example, if the base delay is 1s, and the backoff factor is 2:
 4. delay of 4, retry at t = (3 + 4) = 7
 5. delay of 8, retry at t = (7 + 8) = 15
 
-and so on.  To avoid many clients hammering a server at the same exact time,
+and so on.  To avoid many clients hammering a server at the same time,
```
Comment (theodab) [inline docs/tutorials/network-and-buffering-config.md:34]: I'd keep the "exact". It makes it clearer what the problem is.

File: docs/tutorials/offline.md [hunk 8]
```diff
@@ -331,11 +331,11 @@ the title of the content and the time we downloaded it, but the metadata can
 contain anything you want. The metadata is optional, so you can ignore it if
 you want.
 
-`storage.store` returns an IAbortableOperation that resolves to a
+`storage.store` returns an `IAbortableOperation` that resolves to a
 `shaka.externs.StoredContent` instance (a summary of the stored content).
 
-At this point, the content is now stored offline and it's ready to be played.
-Next we will add functionality to play offline content.
+At this point, the content is now stored offline and's ready to be played.
```
Comment (theodab) [inline docs/tutorials/offline.md:337]: "and's" is not a word.

File: docs/tutorials/ui-customization.md [hunk 9]
```diff
@@ -91,7 +91,7 @@ ui.configure(config);
 An important note: the 'overflow_menu' button needs to be part of the 'controlPanelElements'
 layout for the overflow menu to be available to the user.
 
-The presence of the seek bar and the big play button in the center of the video element can be
+The presence of the seek bar and the big play button in the centre of the video element can be
```
Comment (theodab) [inline docs/tutorials/ui-customization.md:94]: "centre" is the British english spelling, and "center" is the American english spelling. I'd keep it as "center" for consistency, since we use that elsewhere... in a number of places we can't change, too (e.g. the names of CSS property values).

**Review by theodab — CHANGES_REQUESTED** [review]

File: docs/tutorials/a11y.md [hunk 10]
```diff
@@ -10,7 +9,7 @@ to do it. With just a few lines of code, your custom elements can become
 accessible for everyone!
 
 We use [ARIA labels][] to tell screen readers what the button is for.
-Let's take our skip button from the Customization tutorial and add an aria label to it:
+Let's take our skip button from the Customization tutorial and add an `aria-label` to it:
```
[the remark this answers is not in the document]
Comment (joeyparrish) [reply docs/tutorials/a11y.md:12]: @theodab, I believe it's an attribute, rather than a tag.

[the remark this answers is not in the document]
Comment (theodab) [reply docs/tutorials/a11y.md:12]: Ah, right. I used the wrong word, sorry.

File: docs/tutorials/ui-customization.md (hunk 9 above)
Comment (CodeDotJS) [reply docs/tutorials/ui-customization.md:94]: Understood!

File: docs/tutorials/offline.md (hunk 8 above)
Comment (CodeDotJS) [reply docs/tutorials/offline.md:337]: It's a contraction of __*and*__ and __*is*__. I'm not sure how widely it is used though. Will make it "*and is*".

File: docs/tutorials/network-and-buffering-config.md (hunk 7 above)
Comment (CodeDotJS) [reply docs/tutorials/network-and-buffering-config.md:34]: Isn't "exact same" bringing redundancy? 

For example - 

- *squash the commits together*
- __*squash the commits*__

File: docs/tutorials/manifest-parser.md (hunk 6 above)
Comment (CodeDotJS) [reply docs/tutorials/manifest-parser.md:349]: Makes sense. Will update.

File: docs/tutorials/manifest-parser.md (hunk 5 above)
Comment (CodeDotJS) [reply docs/tutorials/manifest-parser.md:160]: Understood!

File: docs/tutorials/license-server-auth.md (hunk 4 above)
Comment (CodeDotJS) [reply docs/tutorials/license-server-auth.md:193]: Yeah, I don't know why I changed it.

File: docs/tutorials/faq.md (hunk 3 above)
Comment (CodeDotJS) [reply docs/tutorials/faq.md:205]: The article also has `adblocking` mentioned, so I think both are the right conventions. If it's not an issue, we can keep adblockers instead of ad blocker or vice versa.

File: docs/tutorials/faq.md (hunk 2 above)
Comment (CodeDotJS) [reply docs/tutorials/faq.md:126]: Oh! I misread the sentence. Will fix.

File: docs/tutorials/drm-config.md (hunk 1 above)
Comment (CodeDotJS) [reply docs/tutorials/drm-config.md:148]: When inanimate objects are kept in perspective, we use *its* instead of *their*.

File: docs/tutorials/a11y.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (CodeDotJS) [reply docs/tutorials/a11y.md:12]: Understood! Will make the changes.

File: docs/tutorials/drm-config.md (hunk 1 above)
Comment (michellezhuogg) [reply docs/tutorials/drm-config.md:148]: Hello @CodeDotJS , thank you so much for your interest in contributing to Shaka Player, and thank you for reading our docs carefully! 

The docs are meant to help our developers understand how Shaka Player works easily and quickly. More than half of our team members are native English speakers, and we had checked very carefully before we published the docs, to make sure the docs are fairly easy to read and understand. At this point with our limited capacity and core projects priorities, it may not be the best use of both our time and energies to double check, explain and arbitrate the spelling/grammar, especially with a few rounds of reviews and updates.

Our original goal to volunteer at GSoC is to help students to gain more tech and coding experience through our project, and potentially become long-term contributors of Shaka Player. We want to invest our time to help the external developer to understand our code base, and finish projects in our code base. So instead of focusing on the grammar in the docs, I encourage you to look into the projects list and maybe enhancements list if you are interested to know more about thee code base. 

If you're cool with that, please take your time and update the PR one final round by addressing @theodab 's comments. Thank you for your understanding!

**Comment by CodeDotJS** [comment]
updated the changes based on the reviews

@michellezhuogg @theodab

File: docs/tutorials/offline.md (hunk 8 above)
Comment (joeyparrish) [reply docs/tutorials/offline.md]: Let us clarify, then, that it's not in use in American English.  We can't make any claims about the way English may be spoken elsewhere, but the American 30-somethings on this team have never seen this particular contraction.

Sometimes even England surprises me, and some of England's former colonies retain old features of English that nobody else uses any more.  So who knows what to expect with this crazy language?  :-)

Comment (CodeDotJS) [reply docs/tutorials/offline.md]: Totally get your point. I have sort of this habit of writing __and's__ for __and is__ & __and has__, depends upon the tense. The main motive was to replace __and it's__ with __and is__, but my weird writing style made me go with the contraction.

File: docs/tutorials/drm-config.md [hunk 11]
```diff
@@ -90,7 +90,7 @@ your manifest.  Use this when you need to confirm that your keys are correct.
 
 #### Clear Key Licenses
 
-If your manifest actually specifies Clear Key, you can also use the normal
+If your manifest specifies Clear Key, you can also use the normal
```
Comment (joeyparrish) [inline docs/tutorials/drm-config.md:93]: Please revert this change

File: docs/tutorials/faq.md [hunk 12]
```diff
@@ -72,7 +72,7 @@ or it may require [extra authentication][auth].
 **not** the HTTPS certificate of the proxy or any files on your proxy.  This
 should be the certificate of the license server given by your provider.
 
-The certificate can only be used for that license server, but can be used with
+The certificate can only be used for that license server but can be used with
```
Comment (joeyparrish) [inline docs/tutorials/faq.md:75]: Please revert

File: docs/tutorials/faq.md [hunk 13]
```diff
@@ -204,10 +204,10 @@ supported.  This supports both DASH and HLS manifests.
 **A:** Are you looking at the uncompiled build with an AdBlocker enabled?
 We're rolling out ad support which is triggering some ad blockers to block
 requests for some of our source files. This only affects the uncompiled build.
-Switch to the compiled build (add "build=compiled" to the url) or temporarily
+Switch to the compiled build (add "build=compiled" to the URL) or temporarily
 disable your ad blocker to see the nightly uncompiled mode.
 Please note that if you want to test our ad logic, you might have to disable
-the ad blocker in compiled mode as well.
+the ad blocker in the compiled mode as well.
```
Comment (joeyparrish) [inline docs/tutorials/faq.md:210]: Please revert

File: docs/tutorials/license-server-auth.md [hunk 14]
```diff
@@ -120,7 +120,7 @@ This endpoint requires a specific cookie to deliver a license.  If you try to
 use it without setting the parameter, you will see `Error code 6007`
 (`LICENSE_REQUEST_FAILED`) just as with the other endpoints.
 
-Cookies are set by a server to be returned to that server, and are not sent by
+Cookies are set by a server to be returned to that server and are not sent by
```
Comment (joeyparrish) [inline docs/tutorials/license-server-auth.md:123]: Please revert

File: docs/tutorials/offline.md [hunk 15]
```diff
@@ -346,7 +346,7 @@ the TODO in “playContent” labeled “play offline content” with:
 window.player.load(content.offlineUri);
 ```
 
-Yes, that is really all there is to play offline content. The player actually
+Yes, that is all there is to play offline content. The player
```
Comment (joeyparrish) [inline docs/tutorials/offline.md:349]: Please revert

File: docs/tutorials/manifest-parser.md [hunk 16]
```diff
@@ -346,8 +346,8 @@ MyManifestParser.prototype.loadReference_ =
 
 If your content is encrypted, there are a few changes to the manifest you need
 to do.  First, for each Stream that contains encrypted content, you need to set
-`stream.encrypted` to true and put the key IDs that the stream is encrypted with
-in the `stream.keyIds` set.  Filling out the `keyIds` is technically optional,
+`stream.encrypted` to true and put the key IDs that the stream is encrypted with in
+the `stream.keyIds` set.  Filling out the `keyIds` is technically optional,
```
Comment (joeyparrish) [inline docs/tutorials/manifest-parser.md:350]: Not sure why you moved the word "in" from one line to another.  Am I overlooking some other change in the diff?

File: docs/tutorials/manifest-parser.md [hunk 17]
```diff
@@ -157,14 +157,14 @@ Media segment times are all in terms of the presentation timeline.  So if the
...
-First we ask for the index that corresponds with a start time.  Then on update,
+First, we ask for the index that corresponds with a start time.  Then on update,
 we increment the index and ask for segments in order. The value of the index
 doesn't matter, but indices must be sequential integers.
 
 #### createSegmentIndex():Promise
 
 This is called first before any other method.  This allows an index to be
-fetched over the network, if needed.  This method should return a Promise that
+fetched over the network if needed.  This method should return a Promise that
```
Comment (joeyparrish) [inline docs/tutorials/manifest-parser.md:167]: Please revert

File: docs/tutorials/drm-config.md [hunk 18]
```diff
@@ -67,7 +67,7 @@ use it.
 The EME spec requires browsers to support a common key system called "Clear
 Key".  *(At the time of this writing (April 2016), only Chrome and Firefox
 have implemented "Clear Key".)*
-Clear Key uses unencrypted keys to decrypt CENC content, and can be useful
+Clear Key uses unencrypted keys to decrypt CENC content and can be useful
```
Comment (joeyparrish) [inline docs/tutorials/drm-config.md:70]: Please revert

File: docs/tutorials/drm-config.md [hunk 19]
```diff
@@ -43,13 +43,13 @@ all you need to do.
 #### Choosing a Key System
 
 Shaka Player is key-system-agnostic, meaning we don't prefer any key systems
-over any others.  We use EME to ask the browser what it supports, and make no
+over any others.  We use EME to ask the browser what it supports and make no
```
Comment (joeyparrish) [inline docs/tutorials/drm-config.md:46]: Please revert

**Review by joeyparrish — CHANGES_REQUESTED** [review]

**Comment by CodeDotJS** [comment]
@joeyparrish done. Reverted the changes. Is everything good now?

File: docs/tutorials/manifest-parser.md (hunk 16 above)
Comment (CodeDotJS) [reply docs/tutorials/manifest-parser.md:350]: Originally, it was "with in", but to me, it seemed like using "within" would be the right approach. Later, @theodab suggested the changes, so I reverted it back to the original state. That's just what I did.

**Comment by michellezhuogg** [comment]
1. Please wrap each line at 80 columns. I use the ruler in my text editor to check that.
2. Please double check the your changes, and only keep the necessary ones. Thank you!

File: docs/tutorials/faq.md [hunk 20]
```diff
@@ -233,7 +233,7 @@ an HLS manifest, we do our best to guess what the codecs might be, but those
 guesses might not always be accurate.  If an HLS manifest has no codec
 information provided, we default to guessing that the video codec is
 `avc1.42E01E` and the audio codec is `mp4a.40.2`, which can cause problems if
-the stream is actually video-only or audio-only.  In this case, you can enable
+the stream is video-only or audio-only.  In this case, you can enable
```
Comment (joeyparrish) [inline docs/tutorials/faq.md:236]: I think this should be reverted, too.  The emphasis of "actually" is meaningful, to show the distinction between what we guessed, and reality.  "We thought it was this, but it was _actually_ that."

File: docs/tutorials/manifest-parser.md (hunk 16 above)
Comment (joeyparrish) [reply docs/tutorials/manifest-parser.md:350]: I think as programmers, we could disambiguate things if English were parenthetical.  For example, this is how you should understand the sentence in your head:

> You need to ((set `stream.encrypted` to true) and (put (the key IDs that the stream is encrypted with) in (the `stream.keyIds` set)).

There is an obnoxious rule in English that you should not end sentences with a preposition.  In practice, of course, people do this all the time.  Here, we have a phrase that ends with a preposition: "the key IDs that the stream is encrypted with".  It could be rephrased as "the key IDs with which the stream is encrypted", if you think that would help with clarity from your perspective.

Another small change that could clarify is that the word "in" could become "into" (as in "into the keyIds set").  If you think that would also help with clarity, I think that would be a reasonable improvement.

We recognize that with developers contributing from all over the world, it is really helpful to have docs that are understandable to non-native English speakers.  So we are definitely motivated by arguments about clarity.  If you didn't understand what we meant here, it should be fixed!  After that, it's just a matter of finding the right fix which makes sense to everyone.

Thanks!

**Review by joeyparrish — CHANGES_REQUESTED** [review]

File: docs/tutorials/faq.md (hunk 20 above)
Comment (CodeDotJS) [reply docs/tutorials/faq.md:236]: We are clearly stating that the stream is __video-only__ or __audio-only__. There's a clear cut distinction, and the sentence is explicit even without *"actually"* because we used "__only__" which it's putting direct emphasis, thus, making "actually" redundant.

Although I don't quite agree with you on this, I can revert the changes if you say so.
